src/nvim/fileio.c

```diff
@@ -5992,18 +6005,20 @@ void ex_doautoall(exarg_T *eap)
   check_cursor();           /* just in case lines got deleted */
 }
 
-/*
- * Check *argp for <nomodeline>.  When it is present return FALSE, otherwise
- * return TRUE and advance *argp to after it.
- * Thus return TRUE when do_modelines() should be called.
- */
-int check_nomodeline(char_u **argp)
+/// Check *argp for <nomodeline>.  When it is present return false, otherwise
+/// return true and advance *argp to after it.
+/// Thus return true when do_modelines() should be called.
+///
+/// @param[in,out] The argument string
```

Comment:
I strongly urge every to omit "the" from parameter the start of descriptions unless it really is unreadable otherwise.